readthedocs-theme/templates/readthedocs/product.html

```diff
@@ -0,0 +1,256 @@
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="content">
+            <h4>Multiple versions</h4>
+            <p>
+              We can host and build multiple versions of your docs so having a
+              1.0 version of your docs and a 2.0 version of your docs is as easy
+              as having a separate branch or tag in your version control system.
+            </p>
+          </div>
+          <div class="extra content">
+            <div class="ui labels">
+              <span class="ui label basic large"><i class="icon primary hourglass outline"></i> Version control</span>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="content">
+            <h4>You Write docs with Sphinx or Mkdocs</h4>
+            <p>
+              Documentation can be written in Markdown or reStructuredText. Write just prose documentation, or automatically pull in documentation that is written inside your code (eg. Python docstrings).
+            </p>
+          </div>
+          <div class="extra content">
+            <div class="ui labels">
+              <span class="ui label basic large"><i class="icon primary hourglass outline"></i> reStructuredText</span>
+              <span class="ui label basic large"><i class="icon primary hourglass outline"></i> Markdown</span>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="content">
+            <h4>We Build your docs on every Pull Request or merge</h4>
+            <p>
+              Whenever you push a commit, we build your full set of documentation and provide a fully working preview. If you decide to merge into a production version, we will update that on merge. Your docs are always up to date with your code.
+            </p>
+          </div>
+          <div class="extra content">
+            <div class="ui labels">
+              <span class="ui label basic large"><i class="icon primary hourglass outline"></i> Always up to date</span>
+              <span class="ui label basic large"><i class="icon primary hourglass outline"></i> Automation</span>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="content">
+            <h4>The same tools your team is already using</h4>
+            <p>
+              All too often, when documentation is lacking, it's because maintaining documentation is too difficult. Using developer tools like git lowers the bar contributing documentation, meaning your docs will get better, faster.
+            </p>
+          </div>
+          <div class="extra content">
+            <div class="ui labels">
+              <span class="ui label basic large"><i class="icon primary hourglass outline"></i> Your own tools</span>
+              <span class="ui label basic large"><i class="icon primary hourglass outline"></i> Docs as code</span>
```

Comment:
"Docs as code" sounds like a fragment, and readers might not be aware of the concept as it's a phrase that came out of the WTD community. Perhaps better to focus on teams collaborating on this bullet point?

Reply:
We talked a little bit about this before.
I see what you mean. "Docs as code" is a bit lost sentence at the moment.

Still I would like to emphasize this idea throughout the website as I believe this is what separates Read the Docs from its competitors:

* Read the Docs treats docs as code and, by this, aims to approach the technical writer teams to the developers.
* Competitors use WYSIWYG.

WYSIWYG is much more flashy and calls for more ostentatious design that is easier to attract user attention. We need to emphasize our differences and strengths too.

Reply:
This is a great point.

"Use your own tools" sort of does that, but I agree that "You don't have to waste your time in a WYSIWG editor" is a separate point entirely.

I don't have any stellar suggestions. Perhaps just expanding this to "Write docs like you write code" or something similar for now?

Reply:
I opened an  issue related to this topic here #31.

Reply:
+1 on "Write docs like you write code" or maybe to shorten it a bit "Write docs like you code".

Reply:
```suggestion
              <span class="ui label basic large"><i class="icon primary hourglass outline"></i> Write docs like you code</span>
```